src/LcdMenu.cpp

```diff
@@ -122,14 +134,18 @@ void LcdMenu::setBacklightBrightness(int level, bool persist)
 {
   _brightness = level;
 
-  //LOGV2(DEBUG_INFO, F("LCD: Writing %d as brightness"), _brightness  );
-  // analogWrite(10, _brightness);
-  //LOGV2(DEBUG_INFO, F("LCD: Wrote %d as brightness"), _brightness  );
+  #if DISPLAY_TYPE == DISPLAY_TYPE_LCD_KEYPAD && defined(LCD_BRIGHTNESS_PIN)
+    analogWrite(LCD_BRIGHTNESS_PIN, _brightness);
```

Comment:
Brightness does not work on a Mega with a standard LCD Shield, we should disable this by default and make the CAL menu entry disable-able through defines.  As soon as you change from 255 down it goes dark, then once you get to around 194 it crashes hard.

Reply:
We really have to investigate the crash at some point ... it should actually work.

Reply:
I have already spent too much time on this, back when I first try to add it... :-)

Reply:
Unfortunately I don't an ATmega to test this. I suggest to comment it out for now and add a TODO mentioning the crash.